models/Sign_up.js

```diff
@@ -0,0 +1,29 @@
+const Sequelize = require('sequelize');
```

Comment:
This is not a model, this is a mapping between model and table.
> To define mappings between a model and a table, use the **define** method

Maybe we need a model, in the manual there is this example:
```js
class Project extends Model {}
Project.init({
  title: Sequelize.STRING,
  description: Sequelize.TEXT
}, { sequelize, modelName: 'project' });
```